n26/core/templates/n26/includes/equip_overspend.html

```
@@ -0,0 +1,53 @@
+page. It carries the dialog host's id and hx-swap-oob, so it replaces
+whatever that host holds — which is how a panel opens.
+
+The same `confirmation` the whole-page n26/confirm.html draws, asked in the
+place a screen that is not being rebuilt can ask it. The figures are the
+component both use, so the two cannot come to disagree about the arithmetic.
+
+`redrawn` tells the panel it arrived on its own: dismissing puts the address
+back rather than navigating away from a page that never reloaded.
+{% endcomment %}
+<div id="n26-dialog-host" hx-swap-oob="true">
+    <c-n26.dialog title="{{ confirmation.title }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)